### ISSUE TITLE
Contrast and accessibility issues.

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/taotnpwaft#98]
From index.html
Some contrast and accessibility issues exist, tested with Firefox 99 accessibility testing in the console/devtools (Linux x64).

- [ ] Text container for the hamburger menu bars show this keyboard accessibility issue

[Clickable elements must be focus-able and should have interactive semantics](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard#clickable_elements_must_be_focusable_and_should_have_interactive_semantics)

If an element can be clicked with a pointing device, such as a mouse, then it should also be focusable using the keyboard, and the user should be able to do something by interacting with it.

An element is clickable if it has an onclick event handler defined. You can make it focusable by adding a tabindex=0 attribute value to it. You can make it operable with the keyboard by defining an onkeydown event handler; in most cases, the action taken by event handler should be the same for both types of events.

- [ ] Contrast issues.  Fix colorization so that the light / dark and the colors also present the best percentage and are WCAG compliant.

Aria Text Label(s)
- [ ] the TAOT logo needs a Aria Text Label.
- [ ] the Enter Email address field in the footer needs a Aria Text Label
- [ ] the dark mode / light mode needs a Aria Text Label
- [ ] the Search bar in the top nav needs a Aria Text Label

The left nav drawer needs some contrast issues fixed.  Perhaps a highlight onClick? make them focus-able?

**Comment by gbowne1** [comment]
This issue was moved as a TODO item to the new Org Repository.
